File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/course_detail/view/CourseDetailView.java
package org.wikiedufoundation.wikiedudashboard.course_detail.view;

import org.wikiedufoundation.wikiedudashboard.course_detail.data.CourseDetail;
import org.wikiedufoundation.wikiedudashboard.helper.Progressive;
import org.wikiedufoundation.wikiedudashboard.helper.Toaster;

public interface CourseDetailView extends Progressive, Toaster {
    void setData(CourseDetail data);
}

File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/course_detail/common/view/CourseDetailView.kt
package org.wikiedufoundation.wikiedudashboard.course_detail.common.view

import org.wikiedufoundation.wikiedudashboard.course_detail.common.data.CourseDetail
import org.wikiedufoundation.wikiedudashboard.helper.Progressive
import org.wikiedufoundation.wikiedudashboard.helper.Toaster

interface CourseDetailView : Progressive, Toaster {
    fun setData(data: CourseDetail)
}
